internal/store/migrations.go

```diff
@@ -921,6 +921,65 @@ var migrations = []migration{
 			return err
 		}
 
+		return nil
+	}},
+	{semver.MustParse("0.18.0"), semver.MustParse("0.19.0"), func(e execer) error {
+		_, err := e.Exec(`ALTER TABLE Cluster RENAME TO ClusterTemp;`)
```

Comment:
I don't think you need to do this. I'm pretty sure you can just do `ALTER TABLE Cluster ADD COLUMN` for each of the columns you need. I could swear I tested this with both Postgres and sqlite and it works with both.

Reply:
Yeah, a total of three columns were added and three were removed (basically a rename) so this felt a little more straightforward this time.